src/components/job/JobList.jsx

```diff
@@ -41,7 +57,15 @@ export default function JobList() {
   }, [data, jobId]);
 
   return (
-    <div className="flex flex-row">
+    <div className="w-full">
```

Comment:
could you remove the search and filter here? Tianchi has started his work and I want to avoid conflict.

Reply:
sure, just removed